README.adoc

```diff
@@ -269,9 +272,9 @@ include::{common-includes}/devmode-build.adoc[]
 
 When the server is running, select either approach to fetch your system properties:
 
-- Visit the http://localhost:9080/inventory/systems/localhost[http://localhost:9080/inventory/systems/localhost^] URL. The URL retrieves the system property information for `localhost` host name by invoking the http://localhost:9080/system/properties[http://localhost:9080/system/properties^] service.
+* Visit the http://localhost:9080/inventory/systems/localhost[http://localhost:9080/inventory/systems/localhost^] URL. The URL retrieves the system property information for `localhost` host name by making a request to the `system` service at http://localhost:9080/system/properties[http://localhost:9080/system/properties^].
```

Comment:
The URL retrieves the system property information for `localhost` host name

--->

The URL retrieves the system property information for the`localhost` host name